src/granite/converters.cr

```diff
@@ -0,0 +1,11 @@
+module Granite::Converters
```

Comment:
What do you think about adding the `to_db` and `from_rs` to the custom class by re-opening it? That way you don't need to specify a converter?

Reply:
In this case you mean reopen `UUID` to add them?

Reply:
Hhhm.  that would work but not sure I like re-opening it.  Another option would be an abstract base class that we would inherit from.

For example:
```
class User < Granite::Base
...
  primary uuid : MyUUID
  field encrypted_password : EncryptedPassword
end

class MyUUID < Granite::CustomType
  property uuid : UUID

  def to_db
    uuid.to_s
  end

  def from_rs(value)
    uuid = UUID.new(value)
  end
end

class EncryptedPassword < Granite::CustomType
  property password : String

  def to_db
    //encrypt before storing
  end

  def from_rs
    //decrypt after retrieval???
  end
end
```

Just throwing out some ideas here.  Let me know what you think.

Reply:
Hmm, I dunno.  That seems like a lot more work than just adding a `, converter: Granite::Converters::UuidConverter` to the field.